packages/trpc/server/routers/loggedInViewer/outOfOffice.handler.ts

```diff
@@ -25,7 +25,7 @@ export const outOfOfficeCreate = async ({ ctx, input }: TBookingRedirect) => {
 
   const inputStartTime = dayjs(startDate).startOf("day");
   const inputEndTime = dayjs(endDate).endOf("day");
-  const offset = dayjs(inputStartTime).utcOffset();
+  const offset = input.offset;
```

Comment:
Damn, e2e test is real. This LOC culprit makes me laugh hard :rofl:

Reply:
Let me check it one more time to be sure.

Reply:
@anikdhabal I confirmed this issue consists of a flicker issue and that time start timezone offset issue. So these 2 fixes are required to enable these tests.

Reply:
yes, please